.github/workflows/build_and_push.yml

```diff
@@ -247,3 +191,14 @@ jobs:
             DBUILD_REPO_URL=https://github.com/openebs/lvm-localpv
             DBUILD_SITE_URL=https://openebs.io
             BRANCH=${{ env.BRANCH }}
+
+  release-chart:
+    runs-on: ubuntu-latest
+    needs: ["csi-driver"]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Publish lvm localpv develop or prerelease helm chart
+        uses: stefanprodan/helm-gh-pages@master
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          charts_dir: ./deploy/helm
```

Comment:
Extra line please?

Reply:
done.
Also added crds deletion in test script's cleanup function.